test_unstructured/documents/structured_jsons/example.json

```diff
@@ -0,0 +1,121 @@
+        },
+        "text": "From field name",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "59242237-3ed7-41b6-8a07-7e2003f8ed81",
+        "metadata": {
+            "category_depth": 2,
+            "page_number": 1,
+            "parent_id": "3a6b156a-8176-4e17-be12-8264241f8136",
+            "text_as_html": "<section class=\"Section\" id=\"59242237-3ed7-41b6-8a07-7e2003f8ed81\" />"
+        },
+        "text": "",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "dc3792d4-422e-444f-9087-6b56d0cfb20d",
+        "metadata": {
+            "category_depth": 3,
+            "page_number": 1,
+            "parent_id": "59242237-3ed7-41b6-8a07-7e2003f8ed81",
+            "text_as_html": "<table class=\"Table\" id=\"dc3792d4-422e-444f-9087-6b56d0cfb20d\"> <thead class=\"TableHeader\" id=\"50a5548a-87e8-4024-af59-0b3d2830d140\"> <tr class=\"TableRow\" id=\"5e473d77-4247-4412-be72-dc4e2c45bd4a\"> <th class=\"TableCellHeader\" id=\"01800309-aa42-411c-98ae-30f85b23f399\">Description </th><th class=\"TableCellHeader\" id=\"c2765b63-d089-46a2-8519-55e79e301de4\">Row header </th></tr></thead><tbody class=\"TableBody\" id=\"e0a9a8ff-dd71-48ad-8b4a-274b073d340a\"> <tr class=\"TableRow\" id=\"77e82997-4632-4551-9133-0b0b8545d1e3\"> <td class=\"TableCell\" id=\"7fee12d4-c555-4b7d-a778-d6f8fdec8a57\">Value description </td><td class=\"TableCell\" id=\"5a7a33b0-c57b-4eb8-81a3-5bce9f87c831\"> <span class=\"Currency\" id=\"87220f9d-62c3-482e-92e7-de72a26869cd\">50 $ </span><span class=\"Measurement\" id=\"0095b9ef-b90a-4cca-991e-73547c7165f1\">(1.32 %) </span></td></tr></tbody></table>"
+        },
+        "text": "Description Row header   Value description  50 $ (1.32 %)",
+        "type": "Table"
+    },
+    {
+        "element_id": "1032242a-f75c-4b37-984e-a7fea9aac74c",
+        "metadata": {
+            "category_depth": 2,
+            "page_number": 1,
+            "parent_id": "3a6b156a-8176-4e17-be12-8264241f8136",
+            "text_as_html": "<section class=\"Section\" id=\"1032242a-f75c-4b37-984e-a7fea9aac74c\" />"
+        },
+        "text": "",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "2a4e2c4a-689f-4f9a-8c18-0b6b521e45c3",
+        "metadata": {
+            "category_depth": 3,
+            "page_number": 1,
+            "parent_id": "1032242a-f75c-4b37-984e-a7fea9aac74c",
+            "text_as_html": "<h2 class=\"Subtitle\" id=\"2a4e2c4a-689f-4f9a-8c18-0b6b521e45c3\">2. Subtitle </h2>"
+        },
+        "text": "2. Subtitle",
+        "type": "Title"
+    },
+    {
+        "element_id": "5591f7a4-df01-447e-8251-5ce45f686fbe",
+        "metadata": {
+            "category_depth": 3,
```

Comment:
this `category_depth` should be `1` because there are no `NarrativeText` blocks that this `NarrativeText` block is nested under (which generally may never happen)

Reply:
*this does not have to be a blocker for this PR, can be tracked as follow on

Reply:
`category_depth` only counts the nested depth of the _same_ type of tags or elements and ignores all others in between -- the word `category` in `category_depth` is the cue for "same category" (or type/tag).
"layout" elements don't really matter here, i think.
in reality, there are not many cases where this matters in HTML. mainly list items, heading tags, and once in awhile tables in HTML.